common/src/main/java/io/netty/util/internal/logging/CommonsLoggerFactory.java

```diff
@@ -26,9 +26,12 @@
  * <a href="http://commons.apache.org/logging/">Apache Commons Logging</a>
  * logger.
  */
-public class CommonsLoggerFactory extends InternalLoggerFactory {
+public final class CommonsLoggerFactory extends InternalLoggerFactory {
 
-    Map<String, InternalLogger> loggerMap = new HashMap<String, InternalLogger>();
+    public static final InternalLoggerFactory INSTANCE = new CommonsLoggerFactory();
+
+    private CommonsLoggerFactory() {
```

Comment:
@windie we need to keep the public constructor to not break things. That said we should mark it `@deprecated` and let the user know that the singleton field should be used.